apps/storefront/pages/god-praksis/tilgjengelighet/kontrast.mdx

```diff
@@ -0,0 +1,181 @@
+        </List.Item>
+      </List.Unordered>
+    </List.Root>
+  </Card.Content>
+</Card>
+
+<br />
+
+<Card color='second'>
+  <Card.Content>
+    <List.Root>
+      <List.Heading
+        level={3}
+        size='xsmall'
+      >
+        Fremtidig eller strengere:
+      </List.Heading>
+      <List.Unordered>
+        <List.Item>
+          **1.4.6 Kontrast** (forbedret) (Nivå AAA): Den visuelle presentasjonen
+          av tekst og bilder av tekst har et kontrastforhold på minst 7:1,
+          unntatt uvesentlig tekst og skriftstørrelser større enn 18px eller
+          14px fet. [Retningslinjer for universell utforming av nettinnhold
+          (WCAG) 2.1
+          (w3.org)](https://www.w3.org/Translations/WCAG21-no/#contrast-enhanced)
+        </List.Item>
+        <List.Item>
+          **WCAG 2.2: 2.4.13** Focus Appearance (Nivå AAA), om utseende til
+          fokusmarkering krever at fokusindikator har en kontrastverdi på 3:1
+          mellom samme piksler i fokusert og ikke-fokusert tilstand.
+          [Understanding Success Criterion 2.4.13: Focus Appearance | WAI |
+          W3C](https://www.w3.org/WAI/WCAG22/Understanding/focus-appearance.html)
+        </List.Item>
+        <List.Item>
+          **WCAG 3** har et krav om farge og kontrast, visuell kontrast i tekst
+          (sølv): Sørg for tilstrekkelig kontrast mellom tekst i forgrunnen og
+          bakgrunnen for teksten. Her brukes det en ny metode, med navn APCA,
+          for å regne ut kontrasten.
+        </List.Item>
+      </List.Unordered>
+    </List.Root>
+  </Card.Content>
+</Card>
+
+## Ny metode utforskes
+
+En ny metode som er mer presis enn dagens standard for å kalkulere kontrast og sette terskelverdier, utforskes nå i WCAG 3.0.
+
+<List.Root>
+  <List.Unordered>
+    <List.Item>
+      Metoden forbedrer hvordan verdien mellom to farger bestemmes, og gjør også
+      forskjell på om fargene er i forgrunnen eller i bakgrunnen.
+    </List.Item>
+    <List.Item>
+      Den setter også tydelige terskelverdier eller målverdier for valg av font,
+      tekststørrelse og font-vekt. Metoden heter Advanced Perceptual Contrast
+      Algorithm (APCA).
+    </List.Item>
+    <List.Item>
+      Målet vårt er å ligge over AAA-krav i WCAG 2.1, og vi vil dermed ligge
+      nærmere terskelverdiene i APCA. Det øker sjansen for at vi skal oppfylle
+      kravet om at alle, også svaksynte, skal kunne se innholdet på nettstedet.
+    </List.Item>
+  </List.Unordered>
+</List.Root>
+
+## Vi bruker i dag en ønsket standard
+
+For at innholdet i offentlige, digitale tjenester skal være tilgjengelig for brukerne, også de med svekket syn, bruker vi i dag en ønsket standard for kontrast, som er høyere enn kravene som stilles til tilgjengelighet. Standarden heter WCAG 2.1 krav på nivå AAA.
+For å øke lesbarheten ytterligere, er ambisjonen vår å oppnå kontrast på nivå 4, som er toppnivået i APCA-metoden, som nå blir utforsket i WCAG 3.0.
```

Comment:
For å øke lesbarheten enda mer, har vi ambisjoner om å oppnå kontrast på nivå 4. Det er toppnivået i APCA-metoden, som nå blir foreslått i WCAG 3.0